src/App.vue

```diff
@@ -3,9 +3,25 @@
     <router-view/>
   </div>
 </template>
+
 <script>
+import Vue from 'vue';
+import VueHead from 'vue-head';
+import openSearchDescription from '@/xml/open_search_description.xml';
+
+Vue.use(VueHead);
+
 export default {
   name: 'App',
+  head: {
+    link: [
+      {
+        rel: 'search',
+        type: 'application/opensearchdescription+xml',
+        href: openSearchDescription,
+      },
```

Comment:
Sorry for picking on this only now, but instead of adding this at runtime, isn't it possible to add the `<link>` tag to the index.html page during build time, just like the script and css assets?

Reply:
If you run `npm run build` you will see that the CSS and JS assets are included in the `dist/index.html` file. The open search html link should probably be added in the same way

Reply:
I'll need to look into it. I can't recall but must've been some reason I had to take this convoluted path.